.github/workflows/build-and-publish.yml

```diff
@@ -3,12 +3,20 @@ name: Build and Publish
 on:
   workflow_call:
     inputs:
+      app_name:
+        description: "name of application folder under infra directory"
+        default: frontend
```

Comment:
i'd say if we are going to remove the default in the Makefile we shouldn't have a default here either